test/scenarios/istio/istio_test.go

```diff
@@ -44,20 +44,46 @@ func TestMain(m *testing.M) {
 		testEnv.Finish(operator.UninstallViaMake(true))
 		testEnv.Finish(namespace.CreateForEnv(nsWithoutIstio))
 	}
+
+	testEnv.AfterEachTest(func(ctx context.Context, c *envconf.Config, t *testing.T) (context.Context, error) {
+		if t.Failed() {
+			// Log events if the test failed
```

Comment:
(IMO) the docstring is a bit unnecessary, the whole things is more readable without it :D

Reply:
Removed, it was added by AI :( usually more like "capitan obvious"

removed in 645bfab44